site/pages/retirement/[beneficiary_address]/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { GetStaticProps } from "next";
+
+import { IS_PRODUCTION } from "lib/constants";
+
+import { getRetirements } from "@klimadao/lib/utils";
+import { RetirementPage } from "components/pages/Retirement";
+import { loadTranslation } from "lib/i18n";
+
+export const getStaticProps: GetStaticProps = async (ctx) => {
+  try {
+    if (IS_PRODUCTION) {
+      throw new Error("Not on Staging");
+    }
+
+    const { params, locale } = ctx;
+
+    if (!params || !params?.beneficiary_address) {
+      throw new Error("No params found");
+    }
```

Comment:
as I later cleanup you can pass a type argument to GetStaticProps for the expected params

Reply:
Will do in follow-up 👍